list.php

```diff
@@ -0,0 +1,56 @@
+<?php
+require "header.php";
+?>
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+<meta charset="UTF-8">
+<title>List</title>
+<body>
+	<div align="center">
+		<form action="register.php" method="post">
+			<input type="submit" value="新規登録" />
+		</form>
+	</div>
+<?php
+$sql = 'SELECT * FROM questions';
+$stmt = $db->prepare($sql);
+$stmt->execute();
+$q_list = $stmt->fetchAll(PDO::FETCH_ASSOC);
+foreach ($q_list as $q_value) {
+    ?>
+<label for="question_id">問題：<?= $q_value['id'] ?></label>
+	<input type="text" name="question" id="question_id"
+		value="<?= $q_value["question"]; ?>" />
+	<br />
+<?php
+    $sql = 'SELECT * FROM correct_answers';
+    $stmt = $db->prepare($sql);
+    $stmt->execute();
+    $ca_list = $stmt->fetchAll(PDO::FETCH_ASSOC);
+    foreach ($ca_list as $ca_value) {
+        if ($ca_value['id'] == $q_value['id']) {
+            ?>
+<label for="answer_id">答え：<?= $ca_value['id'] ?></label>
+	<input type="text" name="answer" id="answer_id"
+		value="<?= $ca_value["answer"]; ?>" />
+	<br />
+	<form action="edit.php" method="post">
+		<input type="hidden" name="question_id"
+			value="<?= $ca_value["id"]; ?>" /> <input type="submit" value="編集" />
```

Comment:
name="question_id"なので
```suggestion
			value="<?= $q_value['id']; ?>" /> <input type="submit" value="編集" />
```